lib/count_betting_and_lottery_events/main.py

```diff
@@ -2,40 +2,38 @@
 import asyncio
 import os
 from datetime import datetime, timedelta
+from re import I
 
 import pandas as pd  # type: ignore
-from typing import Tuple
 
 from pngme.api import AsyncClient
 
 
 async def get_count_betting_and_lottery_events(
-    api_client: AsyncClient, user_uuid: str, utc_time: datetime
-) -> Tuple[int, int, int]:
+    api_client: AsyncClient,
+    user_uuid: str,
+    utc_starttime: datetime,
+    utc_endtime: datetime,
+) -> int:
     """
     Count events labeled with BettingAndLottery across all institutions
-    over the following date ranges: last 30 days, 31-60 days and 61-90 days.
 
     Args:
         api_client: Pngme Async API client
         user_uuid: the Pngme user_uuid for the mobile phone user
-        utc_time: the time-zero to use in constructing the 0-30, 31-60 and 61-90 windows
-
+        utc_starttime: the UTC time to start the time window
+        utc_endtime: the UTC time to end the time window
     Returns:
         count of BettingAndLottery events within the given time window
     """
-    label = "BettingAndLottery"
-
     institutions = await api_client.institutions.get(user_uuid=user_uuid)
-
-    utc_starttime = utc_time - timedelta(days=90)
     inst_coroutines = [
         api_client.alerts.get(
             user_uuid=user_uuid,
             institution_id=institution.institution_id,
             utc_starttime=utc_starttime,
-            utc_endtime=utc_time,
-            labels=[label],
+            utc_endtime=utc_endtime,
+            labels=["BettingAndLottery"],
```

Comment:
I gave this a thought for some time. I tried to empathise with an average user who is not so into "good" programming and I my conclusion was that the fewer trivial variables we had, the easier to read it would be.

For this case, as it has a clear param name `labels` and the labels are basic and readable strings, having a proxy variable seemed to me as "harder" to follow.